packages/x-components/src/x-modules/next-queries/components/next-queries.vue

```diff
@@ -118,13 +158,31 @@ You can use your custom implementation of the Next Query's content. In the examp
 using the default Next Query's content, an icon is added, as well as a span with the query of the
 Next Query suggestion.
 
-```vue
-<NextQueries>
-  <template #suggestion-content="{suggestion}">
-    <img src="./next-query-icon.svg" class="x-next-query__icon"/>
-    <span class="x-next-query__query">{{ suggestion.query }}</span>
-  </template>
-</NextQueries>
+```vue live
+<template>
+  <div>
+    <SearchInput />
+    <NextQueries>
+      <template #suggestion-content="{ suggestion }">
+        <img src="/assets/icons/bulb.svg" class="x-next-query__icon" />
```

Comment:
Change this icon for the BulbOn, the one from the previous comment.

Reply:
I am trying to add more specific icons, matching this component, i added `Nq1Icon`.